appeals-service-api/src/docs/components.yml

```diff
@@ -5,11 +5,83 @@ components:
       properties:
         uuid:
           type: string
-        test:
+        original-appellant:
+          type: boolean
+        appellant-name:
```

Comment:
It's not specifically related to this PR but sometimes I see sometimes appelant and sometimes applicant.